plugins/kotlin/uast/uast-kotlin-fir/testData/legacyRenderLog/ParameterPropertyWithAnnotation.log.fir.txt

```diff
@@ -8,9 +8,7 @@ UFile (package = )
         UField (name = bar)
             UAnnotation (fqName = org.jetbrains.annotations.NotNull)
         UMethod (name = getBar)
-            UAnnotation (fqName = MyAnnotation)
```

Comment:
Hoping it will not break users code

Reply:
No worries. I guess you left a comment on this one: https://github.com/JetBrains/intellij-community/pull/1700/commits/36ea9a579beb14a61c4e1004849ec65329aaddcc#diff-d857157f41e597027576e1886cbb8ec73f06cd51ffa0f720c1947865a82bbf0d where annotation on property accessor is discarded if no user-site is specified (from FIR LC side: https://kotlin.jetbrains.space/p/kotlin/reviews/183/). This matches FE 1.0 LC: https://github.com/JetBrains/intellij-community/blob/kt-212-master/plugins/kotlin/uast/uast-kotlin-fir/testData/legacyRenderLog/ParameterPropertyWithAnnotation.log.fe10.txt#L10

After that, with one more fix in FE 1.0 LC side this time (avoiding `@null` nullability annotation: https://kotlin.jetbrains.space/p/kotlin/reviews/185/), for this test case---`ParameterPropertyWithAnnotation`, render/log from FE 1.0 UAST and FIR UAST are identical: https://github.com/JetBrains/intellij-community/pull/1704/commits/67c8d0d744217a93712127a3c748da523e3d871e#diff-2e6af7a20b33645b7c056f567552ff90fea1ccdd8594462f737633f1924b7820